src/Bot.spec.ts

```diff
@@ -1,8 +1,8 @@
 import { afterEach, beforeEach, describe, expect, it, vi } from 'vitest'
 
-import { Bot } from '../../src/Bot.js'
+import { Bot } from './Bot.js'
 
-vi.mock('../../src/config.js')
+vi.mock('../config.js')
```

Comment:
Should this be `./config.js`?

```suggestion
vi.mock('./config.js')
```

Reply:
yes, thank you krub!